src/main/target/STM32_UNIFIED/target.h

```diff
@@ -239,6 +239,13 @@
 #define USE_ACCGYRO_LSM6DSO
 #define USE_ACCGYRO_BMI270
 
+#if defined(STM32F405) || (STM32H743)
+#define USE_GYRO_SPI_ICM42605
+#define USE_ACC_SPI_ICM42605
+#define USE_GYRO_SPI_ICM42688P
+#define USE_ACC_SPI_ICM42688P
+#endif
+
```

Comment:
what the reason for not moving those into https://github.com/betaflight/betaflight/pull/11118/files#R329 ?

Reply:
Didn't know about this flag. Very handy!

Reply:
I added these definetions on there before,  then @haslinghuis said I should use https://github.com/betaflight/betaflight/pull/11118#issuecomment-995397367.  Who's really in charge here???

Reply:
@MATEKSYS don't worry. We need multiple member approvals on firmware here.

Reply:
@mikeller

Reply:
@DusKing1 please decide. Do we add this generally to > 512KB targets or just the specific ones which are targeted.
I like to have to have this specific to the targets where this ICM is used. We can always extend or move this later.

Reply:
My point is, as this is a relatively new sensor that might be one of the main stream sensors that widely used by vendors, we should included them in all unified target as possible. As the driver for this gyro sensor only takes round 1.3KB, I prefer get it to all unified target after those flash-saving PR get merged.